models/DataObject/Service.php

```diff
@@ -2022,4 +2003,16 @@ protected static function getInheritedData(Concrete $object, string $key, string
 
         return self::getInheritedData($parent, $key, $requestedLanguage);
     }
+
+    public static function useInheritedValues(callable $fn, bool $inheritValues, array $fnArgs = []): mixed
```

Comment:
What do you think about swapping the first two arguments?

I think it improves the understanding of what it does if the boolean that indicates whether inheritance is enabled or disabled is first, otherwise you might have to scan a bunch of code (the callable block) before you see whether it's on or off.

Also, I think it's better if the callable and its arguments are next to each other.

```suggestion
    public static function useInheritedValues(bool $inheritValues, callable $fn, array $fnArgs = []): mixed
```

Reply:
Agreed, improves the readability 👍